content/Proposals/Accepted/202106-handbook.md

```diff
@@ -0,0 +1,133 @@
+* Also primary writer.
+1. External Teams and Red Hat.
+2. Outside people, contributors to our projects, potential future hires, people interested in best practices, team processes etc.
+
+## Non-Goals
+
+* Support other formats than `Markdown,` e.g. Asciidoc
+* Precise design proposal process (it will come in a separate proposal).
+* Sharing Team Statuses, we use JIRA and GH issues for that.
+
+## How
+
+The idea is simple:
+
+Let's make sure we maintain the process of adding/editing documentation **as easy and rewarding** as possible. This will increase the chances team members will document things more often and adopt this as a habit. Produced content will be more likely complete and up-to-date, increasing the chances it will be helpful to our audience, which will reduce the meeting burden. This will make writing docs much more rewarding, which creates a positive loop.
+
+I propose to use git repository [`handbook`](https://github.com/rhobs/handbook) to put all related team documentation pieces there. Furthermore, I suggest reviewing all documents with a similar flow as code and placing information in the form of markdown files that can be read from both GitHub UI and automatically served on https://rhobs-handbook.netlify.app/ website.
+
+Pros:
+* Matches our [goals](#goals).
+* Sharing by default.
+* Low barriers to write documents in a consistent format, low barrier to consume it.
+* Ensures high quality with local CI and review process.
+
+Cons:
+* Some website maintenance is needed, but we use the same and heavily automated flow in Prometheus-operator, Thanos, Observatorium websites etc.
+
+The idea of a handbook is not new. Many organizations do this e.g [GitLab](https://about.gitlab.com/handbook/handbook-usage/#wiki-handbooks-dont-scale).
+
+> NOTE: The website style might be not perfect (https://rhobs-handbook.netlify.app/). Feel free to propose issues, fixes to the overall look and readability!
+
+### Flow of Adding/Consuming Documentation to Handbook
+
+![flow](../../assets/handbook-proposal.png)
+
+If you want to add or edit markdown documentation, refer to [our technical guide](../../contributing.md).
+
+## Alternatives
+
+1. Organize Team Google Drive with all Google docs we have.
+
+Pros:
+* Great for initial collaboration
+
+Cons:
+* Inconsistent format
+* Hard to track approvers
+* Never know when the doc is "completed."
+* Hard to maintain over time
+* Hard to share and reuse outside
+
+1. Create Red Hat scoped only, a private handbook.
+
+Pros:
+* No worry if we share something internal?
+
+Cons:
+* We don't have many internal things we don't want to share at the current moment. All our projects and products are public.
+* Sharing means we have to duplicate the information, copy it in multiple places.
+* Harder to share with external teams
+* We can't use open source tools, CIs etc.
+
+## Action Plan
```

Comment:
It would be great to have a timeline until when the docs would be moved if the proposal is approved. Should this be included in the roadmap for Q3?

In addition to that, do we know where to find existing documentation? Does it only exist in google drive so far, and what is the exact content that needs to be moved to the handbook?

Reply:
Good point, let me add this info:

* when
* where to find now
* how to extract